--- a/quectel_MHI/src/core/mhi_init.c	2026-01-23 13:44:20
+++ b/quectel_MHI/src/core/mhi_init.c	2026-02-09 10:31:25
@@ -644,7 +644,6 @@
 static const struct file_operations mon_fops_text_u = {
 	.owner =	THIS_MODULE,
 	.open =		mon_text_open,
-	.llseek =	no_llseek,
 	.read =		mon_text_read_u,
 	.release =	mon_text_release,
 };
@@ -2369,7 +2368,7 @@
 }
 
 /* match dev to drv */
-static int mhi_match(struct device *dev, struct device_driver *drv)
+static int mhi_match(struct device *dev, const struct device_driver *drv)
 {
 	struct mhi_device *mhi_dev = to_mhi_device(dev);
 	struct mhi_driver *mhi_drv = to_mhi_driver(drv);
